submissions/nallikaea/js-dom/index.js

```diff
@@ -0,0 +1,143 @@
+const gryby = [
```

Comment:
Let's make your code accessible to all developers (not only from Ukraine) and create variable names in English. I think `mushrooms` would be much better here